### DISCUSSION TITLE
现在应该是可以支持自定义UA了？

### DISCUSSION THREAD

**Discussion by alili** [tw93/Pake#120] [Ideas]
有蛮多网站需要自定义UA，tauri已经升到了1.2.0，现在可以支持UA了吧

**Comment by tw93** [comment]
可以支持的，现在就支持自定义ua，我在想是不是直接满足和各系统 chrome 一致即可？
此外主要是哪些app，我本地有测试过 飞书啥的可以绕过去

**Reply by alili** [reply]
我觉得直接在文档写一下怎么配置就行，不一定所有人的电脑都装了chrome ...

**Reply by tw93** [reply]
安排  最近版本我弄下
